README.md

```diff
@@ -36,7 +38,7 @@
   - [Send and receive compressed messages](docs/Compressed.md)
   - [Send messages transactionally](docs/TransactionalSend.md)
   - [Send and receive message as CloudEvents](docs/CloudEvents.md) 🆕
-  - [Test components that use RockLib.Messaging](Examples/Example.Application.Testing/) 🆕
+  - [Test components that use RockLib.Messaging](https://github.com/RockLib/RockLib.Messaging/tree/main/Examples/Example.Application.Testing) 🆕
```

Comment:
Why do you need this full link where the docs are relative?

Reply:
I was being lazy and didn't want to modify them when the documentation site processes them. I will change it.

Reply:
So the Examples links you already have processing to handle the full url?  Why not just use the relative path for all links to avoid the extra processing?

Reply:
The absolute url avoids extra processing. I don't have the Examples files in the documentation site. I was cheating by making the links absolute so I wouldn't have to process them.